conf/default.yml

```diff
@@ -30,6 +30,13 @@ requirements:
     command: go version
     type: installed_program
     version: 1.11
+  metasploit:
```

Comment:
@wbooth and I discussed this and decided that since we can support running metasploit remotely via agents that performing the dependency check here gives the wrong impression.  That said, everything else in this PR is solid (e.g. improved error handling, error messaging, and support for optional dependencies). 

@clenk , can you update the PR to remove the metasploit block from the config file but leave the other improvements in place? 

We might revisit this issue later but from the perspective of agents checking for available dependencies in their environment.

Reply:
Done.